esp/clients/ws_dfsclient/ws_dfsclient.cpp

```diff
@@ -716,8 +716,8 @@ IDFSFile *lookupDFSFile(const char *logicalName, unsigned timeoutSecs, unsigned
                     throw makeStringException(-1, "Could not find any DFS services in the target HPCC configuration.");
             }
         }
-        serviceUrl.append(dfsServiceUrls[currentDfsServiceUrl++].c_str());
-        logicalName = remoteLogicalFileName;
+        currentDfsServiceUrl = (currentDfsServiceUrl+1 == dfsServiceUrls.size()) ? 0 : currentDfsServiceUrl+1;
```

Comment:
incidental bug fix, but only affected local case (which is only enabled by a debug option at moment)

Reply:
This isn't equivalent - previously it indexed by the value pre-increment.  Now it is post increment.

Reply:
will swap line 719 and 720.